cmd/translate/main_grpc_integration_test.go

```diff
@@ -90,75 +93,97 @@ func TestMain(m *testing.M) {
 }
 
 // -------------Translation File-------------.
+
+func randUploadData(t *testing.T, schema translatev1.Schema) ([]byte, language.Tag) {
+	t.Helper()
+
+	messagesCount := gofakeit.IntRange(1, 5)
```

Comment:
Alternatively, use `n` name for length. The function scope is very small.

```suggestion
	n := gofakeit.IntRange(1, 5)
```